### ISSUE TITLE
About the train loss

### ISSUE THREAD

**Issue by muzi2045** [sshaoshuai/PointRCNN#116]
I am trying to train the PointRCNN rpn module with lyft dataset, the all classes object points are gather into foreground points to train PointNet++ , but the output train loss make me confused.
it looks like has no decrease trend .
@sshaoshuai 
![image](https://user-images.githubusercontent.com/12984898/66986311-3fd1a800-f0f1-11e9-86bc-4b5134cdd039.png)

**Comment by gakkiri** [comment]
@muzi2045 Same problem, convergence seems to be very slow, stable around 2.x after 200 epochs, and recall is not normal, using all categories.

**Comment by muzi2045** [comment]
same with me, I have trained with 200 epochs, the rpn_loss between 2.x ~3.x
Have you tried using KITTI dataset to train the Pointnet module?
Maybe there are something different in the code.
@gakkiri

**Comment by gakkiri** [comment]
@muzi2045   
I haven't run it on kitti, but this will be my next plan.
In addition, I'm trying training lyft by just using 'car' and excluding all data augmentation, ensure that the model can overfit the dataset. This might indicate to me whether the code works or not.
